doc/classes/Object.xml

```diff
@@ -651,6 +651,7 @@
 				- [code]hint[/code] is [i]how[/i] the property is meant to be edited (see [enum PropertyHint]);
 				- [code]hint_string[/code] depends on the hint (see [enum PropertyHint]);
 				- [code]usage[/code] is a combination of [enum PropertyUsageFlags].
+				[b]Note:[/b] While in GDScript all class members are treated as properties, in C# and some 3rd party languages may be necessary to explicitly mark class members as Godot properties using decorators or attributes.
```

Comment:
```suggestion
				[b]Note:[/b] In GDScript, all class members are treated as properties. In C# and GDExtension, it may be necessary to explicitly mark class members as Godot properties using decorators or attributes.
```

Reply:
Should we clarify "member variables" to avoid any confusion?